package.json

```diff
@@ -25,9 +25,9 @@
   },
   "scripts": {
     "clean-dist": "rimraf ./dist",
-    "clean-printable": "rimraf src/content/**/printable.mdx",
+    "clean-printable": "rimraf --glob src/content/**/printable.mdx",
     "preclean": "run-s clean-dist clean-printable",
-    "clean": "rimraf src/content/**/_*.mdx src/**/_*.json repositories/*.json",
+    "clean": "rimraf --glob src/content/**/_*.mdx src/**/_*.json repositories/*.json",
```

Comment:
It works without glob as well.
<img width="716" alt="Screenshot 2025-01-24 at 6 01 47 AM" src="https://github.com/user-attachments/assets/26ed351f-3e5d-484e-b32e-f19e117d7b3f" />